pom.xml

```diff
@@ -73,6 +73,12 @@ limitations under the License.
 		<version>3.9</version>
 	</dependency>
 
+
```

Comment:
(Comments prefixed by `nit:` means `nitpicking`. These are nice things to have based on the reviewers opinion. You may address by fixing it or may decide to leave it as is. Let me know in chat if it's unclear)

nit: Extra line. It's enough to have 1 new line between consecutive <dependency> sections.